UPGRADING.md

```diff
@@ -1,6 +1,20 @@
 Upgrading
 =========
 
+## 4.1 to 4.2
+
+4.2.0 adds support for reporting C/C++ crashes in Android code. If you are using
+the Gradle build system to export your app, you will need to add the following
+line to your exported project's build.gradle to enable this functionality:
+
+```groovy
+dependencies {
+  // ...
+  // add this line:
+	compile(name: 'bugsnag-android-ndk-release', ext:'aar')
```

Comment:
Should this be `implementation(name: 'bugsnag-android-ndk-release', ext:'aar')`, as `compile` is deprecated?

Reply:
Its both; older versions of unity export with an older gradle version by default, I'll add a note to switch based on whether gradle 4.2+ is used.